scripts/debugScss.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file debugScss.js
+ * @description Compiles scss directly with node-sass for debugging purposes.
+ */
+
+const path = require('path');
+const sass = require('node-sass');
+
+sass.render(
```

Comment:
Do we need a separate debug task for this or can it be incorporated into our webpack build?

Reply:
It's not really a required step of any build process though. It's more like an optional side step, if you happened to be getting mysterious Sass compilation errors like I was but postCSS was obfuscating them. 

If we added this (even a part of it) to the webpack build, it would probably increase compile time by a lot.

Reply:
Ah, I thought perhaps there was an option for `sass-loader` that could provide a custom formatted / verbose compilation error similar to this so we could also catch these errors in development

Reply:
@SimonFinney That's just it -- the errors in development were not helpful 😅